src/composable/chat/useChatHighlights.ts

```diff
@@ -22,6 +22,7 @@ export interface HighlightDef {
 
 	color: string;
 	label: string;
+	isBlocked: boolean;
```

Comment:
For simplicity make this optional

```suggestion
	isIgnored?: boolean;
```